src/styles/views/UserHand.scss

```diff
@@ -33,6 +33,7 @@
   }
 
   &.card-list {
+    margin-top: 0;
```

Comment:
@serainaschraff This fixes that 
![image](https://user-images.githubusercontent.com/75019468/169899380-01f07c31-edb7-4a22-a458-01d486d29bc8.png)

Reply:
great :)